base/operators.jl

```diff
@@ -68,12 +68,16 @@ Supertype operator, equivalent to `T2 <: T1`.
 """
     supertype(T::DataType)
 
-Return the supertype of DataType `T`.
+Return the supertype of type `T`.
+`T` can be a `DataType` or a `UnionAll` type.
```

Comment:
Actually, this sentence isn't very useful, given that the allowed argument type should be indicated in the signature above anyway.

Reply:
Although it could be good to xref `DataType` and `UnionAll`. Like this:

```md
[`DataType`](@ref)
```

```md
[`UnionAll`](@ref)
```